flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -723,8 +723,11 @@ public TableConfig getConfig() {
 
 	@Override
 	public JobExecutionResult execute(String jobName) throws Exception {
-		translate(bufferedModifyOperations);
-		bufferedModifyOperations.clear();
+		try {
```

Comment:
add some comments in `TableEnvironment#execute` to explain the buffer clear behavior

Reply:
I think it's better add the comments to the `TableEnvironment` interface which could help the users to understand the behavior from API level.